azkaban-common/src/main/java/azkaban/executor/ExecutorManager.java

```diff
@@ -453,11 +453,14 @@ private void loadQueuedFlows() throws ExecutorManagerException {
   }
 
   /**
+   * <pre>
```

Comment:
I don't think that is necessary here because it is just a paragraph and not code. `<pre>` is used displaying code blocks within in your documentation: http://docs.oracle.com/javase/7/docs/technotes/tools/windows/javadoc.html

Also, AFAIU, using `<pre>` here will not render the Javadoc tags within this documentation, such as `{@inheritdoc}` because they are treated as code and will be displayed as such.